### PR TITLE
Changing URL at work with the map

### PR DESCRIPTION
**Pull request by Riko196**

### CODE REVIEW

File: client/src/components/Notices/CompaniesTable.scss [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-  background-color: white;
+  background-color: $component-background;
   padding: 0 0 0 20px;
   border-radius: 1px;
   margin-bottom: 0;
 }
 
 .companies-table-header tr th {
-  color: #6c8294; // TODO: Move to variables
+  color: $secondary;
```
Comment (Siegrift) [inline client/src/components/Notices/CompaniesTable.scss]: :+1:

File: client/src/components/Verejne/Map/index.js [hunk 2]
```diff
@@ -48,7 +53,7 @@ const Map = ({
 }: Props) => {
   return (
     <div className="google-map-wrapper">
-      <GoogleMap center={center} zoom={zoom} onChange={onChange}>
+      <GoogleMap center={center} zoom={zoom} onChange={debounce(2000, onChange)}>
```
Comment (Siegrift) [inline client/src/components/Verejne/Map/index.js]: Use https://lodash.com/docs/4.17.10#debounce instead of the dependency

File: client/src/components/Notices/CompaniesTable.scss (hunk 1 above)
Comment (mpinter) [reply client/src/components/Notices/CompaniesTable.scss]: sorry but no

don't create new color variables, there's plenty of colors defined already - one of the existing should fit the purpose well

even if you really must define a new one and don't see other way around it, don't call a color used exactly once secondary

File: client/src/components/Notices/Company.scss [hunk 3]
```diff
@@ -2,27 +2,23 @@
...
 
-.company-title {
-  a {
-    display: block;
-    max-width: 25vw;
-    text-overflow:ellipsis;
-    overflow:hidden;
-    white-space:nowrap;
-  }
+.nowrap-ellipsis {
```
Comment (mpinter) [inline client/src/components/Notices/Company.scss]: @darjanin can I leave css review on you please ?

@Riko196 why this change ?

File: client/src/components/Verejne/Map/index.js [hunk 4]
```diff
@@ -80,11 +85,27 @@ export default compose(
...
   ),
+  withRouter,
   withDataProviders(({entitiesUrl}) => [entitiesProvider(entitiesUrl)]),
   withHandlers({
-    onChange: ({entitiesUrl, setMapOptions, refetch}) => (options) => {
-      setMapOptions(options)
-    },
+    onChange: (props) => (options) => {
+      console.log(props)
+      props.setMapOptions(options)
```
Comment (mpinter) [inline client/src/components/Verejne/Map/index.js]: you definitely don't want to do this debounced - you'll probably want to call a debounced function from inside onChange

File: client/src/components/Verejne/Map/index.js [hunk 5]
```diff
@@ -80,11 +85,27 @@ export default compose(
...
     }
   ),
+  withRouter,
   withDataProviders(({entitiesUrl}) => [entitiesProvider(entitiesUrl)]),
   withHandlers({
-    onChange: ({entitiesUrl, setMapOptions, refetch}) => (options) => {
-      setMapOptions(options)
-    },
+    onChange: (props) => (options) => {
+      console.log(props)
```
Comment (mpinter) [inline client/src/components/Verejne/Map/index.js]: forgotten log

File: client/src/components/Verejne/Map/index.js [hunk 6]
```diff
@@ -80,11 +85,27 @@ export default compose(
...
+      props.setMapOptions(options)
+      props.history.push(
```
Comment (mpinter) [inline client/src/components/Verejne/Map/index.js]: replace instead of pushing , otherwise the person using the app will have to go back through all of the map changes when navigating via back button

File: client/src/components/Verejne/Map/index.js [hunk 7]
```diff
@@ -80,11 +85,27 @@ export default compose(
...
+      )
+    }
+  }),
+  lifecycle({
+    componentDidMount() {
+      const params = qs.parse(this.props.history.location.search.substr(1))
+      this.props.setMapOptions(
+        {center: [Number(params.lat), Number(params.lng)], zoom: Number(params.zoom)}
+      )
+      console.log(this.props)
```
Comment (mpinter) [inline client/src/components/Verejne/Map/index.js]: console
